src/QuartoNotebookWorker/ext/QuartoNotebookWorkerPlotsExt.jl

```diff
@@ -33,12 +33,8 @@ function configure()
         dpi_kwargs = Dict{Symbol,Any}()
     end
 
-    pkgid = Base.PkgId(Plots)
-    if (QuartoNotebookWorker._pkg_version(pkgid) < v"1.28.1") && (fm.fig_format == "pdf")
-        Plots.gr(; size_kwargs..., fmt = :png, dpi_kwargs...)
-    else
-        Plots.gr(; size_kwargs..., fmt = fm.fig_format, dpi_kwargs...)
-    end
+    Plots.gr(; size_kwargs..., dpi_kwargs...)
```

Comment:
Format wasn't really relevant to pass here, since we call `show` on each valid mimetype rather than relying on the default format set here. The addition of `application/pdf` support seems to resolve this.

Reply:
Yeah that's what I gathered from the linked Plots code as well. I'm not sure what backends would actually use pdf assets if we offer them, maybe latex-based ones.